src/libs/portfolio/helpers.ts

```diff
@@ -347,6 +350,44 @@ export const tokenFilter = (
   return isInAdditionalHints || pinnedRequested
 }
 
+const getTokenSafetyLevel = (token: TokenResult): TokenResult['flags']['safetyLevel'] => {
+  let safetyLevel: TokenResult['flags']['safetyLevel'] = 'unknown'
+
+  const hasFlags = token.flags.canTopUpGasTank || token.flags.isFeeToken || token.flags.onGasTank
+
+  if (token.address === ZeroAddress || hasFlags) return 'trusted'
+
+  const canTokenBeChecked = networks.some((network) => network.id === token.networkId)
+
+  // If the token is on a custom network we can't check if it's spoof
+  // thus we can't trust it
+  if (!canTokenBeChecked) return undefined
+
+  Object.values(humanizerInfo.knownAddresses as unknown as HumanizerMeta).some((value) => {
```

Comment:
add comment about `.some()` doing a `break` on `return true` and `continue` on `return false`